packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -44,7 +44,7 @@ function getPaymentMethodTitle(
         const customTitles: { [key: string]: { logoUrl: string; titleText: string } } = {
             [PaymentMethodType.CreditCard]: {
                 logoUrl: '',
-                titleText: methodName,
```

Comment:
I see they are not identical.
```
const methodName = getPaymentMethodName(language)(method);
const methodDisplayName = getPaymentMethodDisplayName(language)(method);
```
Could I know why did you replace it?

Reply:
methodName just grab method name from provider config.
methodDisplayName locally wraps credit card name in translation method.

Reply:
[methodName](https://github.com/bigcommerce/checkout-js/blob/master/packages/payment-integration-api/src/getPaymentMethodName.ts#L100)

        
[ methodDisplayName ](https://github.com/bigcommerce/checkout-js/blob/master/packages/core/src/app/payment/paymentMethod/getPaymentMethodDisplayName.tsx#L33)

Reply:
Hey @vitalii-koshovyi they are still essentially different things. Can we not add text for bluesnap here https://github.com/bigcommerce/checkout-js/blob/master/packages/payment-integration-api/src/getPaymentMethodName.ts#L119. 

As a separate task we can think of how to consolidate the display name/method name into one. But that will require more testing across the app.